collections/index.js

```diff
@@ -72,3 +156,48 @@ export const calculateTotalSalary = () => {
 export const organiseEmployeesByDepartment = () => {
 
 }
+
+const employeeObject = {
+  Engineering: [
+    {
+      "id": 1,
+      "fistName": "Matthew",
+      "lastName": "Tsinontas",
+      "department": "Engineering",
+      "contactNumber": "01611119090",
+      "taxCode": "123AL",
+      "salary": 38000
+    },
+    {
+      "id": 2,
+      "fistName": "Joe",
+      "lastName": "Bloggs",
+      "department": "Engineering",
+      "contactNumber": "",
+      "taxCode": "123AL",
+      "salary": 42000
+    }
+  ],
+  Audio: [
+    {
+      "id": 3,
+      "fistName": "Stevie",
+      "lastName": "Wonder",
+      "department": "Audio",
+      "contactNumber": "01338940929",
+      "taxCode": "108LT",
+      "salary": 28000
+    }
+  ],
+  Accounting: [
+    {
+      "id": 4,
+      "fistName": "Joe",
+      "lastName": "Davidson",
+      "department": "Accounting",
+      "contactNumber": "02289380291",
+      "taxCode": "129BT",
+      "salary": 55000
+    }
+  ]
+}
```

Comment:
Do we still need this here?

Reply:
If I have something to visually deference i'll find it easier.